packages/discovery-provider/src/queries/comments/__init__.py

```diff
@@ -0,0 +1,22 @@
+from src.queries.comments.replies import get_paginated_replies, get_replies
+from src.queries.comments.track_comments import get_track_comments
+from src.queries.comments.user_comments import get_user_comments
+from src.queries.comments.utils import (
+    COMMENT_REPLIES_DEFAULT_LIMIT,
+    COMMENT_ROOT_DEFAULT_LIMIT,
+    get_is_reacted,
+    get_muted_users,
+    get_track_notification_setting,
+)
+
+__all__ = [
```

Comment:
This reminds me I'm not very good at python, as I had to look up what this is for :-)